.github/workflows/CompatHelper.yml

```
@@ -1,17 +1,26 @@
 name: CompatHelper
-
 on:
   schedule:
-    - cron: '0 0 * * *'
-
+    - cron: 0 0 * * *
+  workflow_dispatch:
```

The latest CompatHelper.jl [usage example](https://github.com/JuliaRegistries/CompatHelper.jl/blob/master/.github/workflows/CompatHelper.yml) suggests permission to write.
```suggestion
  workflow_dispatch:
permissions:
  contents: write
  pull-requests: write
```